simtools/applications/validate_camera_efficiency.py

```diff
@@ -23,6 +23,14 @@
         Telescope model name (e.g. LST-1, SST-D, ...)
     model_version (str, optional)
         Model version (default='Current')
+    zenith_angle (float, optional)
```

Comment:
I am commenting here but I want to address the application as a whole:
When running the application without arguments, i.e. `simtools-validate-camera-efficiency`, there is an error:
```
Traceback (most recent call last):
  File "/workdir/env/bin/simtools-validate-camera-efficiency", line 8, in <module>
    sys.exit(main())
  File "/workdir/external/simtools/simtools/applications/validate_camera_efficiency.py", line 129, in main
    tel_model = TelescopeModel(
  File "/workdir/external/simtools/simtools/model/telescope_model.py", line 59, in __init__
    self.site = names.validate_site_name(site)
  File "/workdir/external/simtools/simtools/utils/names.py", line 230, in validate_site_name
    return validate_name(name, all_site_names)
  File "/workdir/external/simtools/simtools/utils/names.py", line 274, in validate_name
    raise ValueError(msg)
ValueError: Invalid name None
```

Instead of an error, I would expect the help message such as this one:

```
(env) simtools-print-array-elements
usage: print_array_elements [-h] --array_element_list ARRAY_ELEMENT_LIST
                            [--compact {corsika,utm,mercator}] [--export {corsika,utm,mercator}]
                            [--use_corsika_telescope_height]
                            [--select_assets SELECT_ASSETS [SELECT_ASSETS ...]]
                            [--data_path DATA_PATH] [--output_path OUTPUT_PATH]
                            [--use_plain_output_path] [--model_path MODEL_PATH]
                            [--simtel_path SIMTEL_PATH] [--config CONFIG] [--env_file ENV_FILE]
                            [--test] [--label LABEL] [--log_level LOG_LEVEL] [--version]
print_array_elements: error: the following arguments are required: --array_element_list
```
Could you please have a look on that (or open an issue to solve it later)?

Reply:
The problem here is that if you do not provide a telescope and site, you get that error instead of the help message. I added a bit of code to protect from that. It's a bit ugly because I check the arguments are provided manually, but I don't know a different way to change an already defined argument to be `required`. Do you know @GernotMaier?

Reply:
OK, so I guess we keep my ugly solution.

Reply:
Yes, I think this solution is fine!